core/storage/question/gae_models.py

```diff
@@ -146,6 +146,27 @@ def create(
 
         return question_model_instance
 
+    @classmethod
+    def put_multi_questions(cls, questions):
+        """Puts multiple question models into the datastore.
+
+        Args:
+            questions: list(Question). The list of question objects
+            to put into the datastore.
+        """
+        cls.put_multi(questions)
+
+
+    @classmethod
+    def delete_multi_questions(cls, questions):
```

Comment:
See above -- feel free to delete this if you're not using it

Reply:
Done